src/libprojectM/CMakeLists.txt

```diff
@@ -1,23 +1,15 @@
 add_compile_definitions(
         DATADIR_PATH="${PROJECTM_DATADIR_PATH}"
-        GL_SILENCE_DEPRECATION
+        $<IF:$<PLATFORM_ID:Darwin>,GL_SILENCE_DEPRECATION,>
+        $<IF:$<PLATFORM_ID:Windows>,NOMINMAX,>
+        $<IF:$<PLATFORM_ID:Windows>,WIN32_LEAN_AND_MEAN,>
+        $<IF:$<PLATFORM_ID:Windows>,STBI_NO_DDS,>
```

Comment:
Only as a note to self, I was just wondering why DDS support was disabled since Milkdrop explicitly supports it. We can keep it disabled for now, but I'll look into that at a later point and find out what the reasons were.